docs: migration guide for register{StandardSchemes => SchemesAsPrivileged} (#16762)
electron/electron: docs/api/protocol.md

are disabled for non standard schemes. So in general if you want to register a
custom protocol to replace the `http` protocol, you have to register it as a standard scheme.

> To replicate the functionality of the previous `registerStandardSchemes` function that existed prior to Electron 5.0.0, for example:

Reviewer: Can we also mention that this replaces the `webFrame.registerURLSchemeAs*` APIs

> To replicate the functionality of the previous `registerStandardSchemes` function that existed prior to Electron 5.0.0, for example:

Reviewer: ```suggestion
To replicate the functionality of the previous `protocol.registerStandardSchemes` and `webFrame.registerURLSchemeAs*` functions that existed prior to Electron 5.0.0, for example:
```

```
# before (<= v4.x)

> protocol.registerStandardSchemes(['scheme1', 'scheme2'], { secure: true })

Reviewer: ```suggestion
// Main
protocol.registerStandardSchemes(['scheme1', 'scheme2'], { secure: true })
// Renderer
webFrame.registerURLSchemeAsPrivileged('scheme1', { secure: true })
webFrame.registerURLSchemeAsPrivileged('scheme2', { secure: true })
```

are disabled for non standard schemes. So in general if you want to register a
custom protocol to replace the `http` protocol, you have to register it as a standard scheme.

> To replicate the functionality of the previous `protocol.registerStandardSchemes`, `webFrame.registerURLSchemeAs*` and `protocol.registerServiceWorkerSchemes` functions that existed prior to Electron 5.0.0, for example:

Reviewer 2: This sentence reads awkwardly.  Could we say something like:
```suggestion
`protocol.registerSchemesAsPriviliged` can be used to replicate the functionality of the previous `protocol.registerStandardSchemes`, `webFrame.registerURLSchemeAs*` and `protocol.registerServiceWorkerSchemes` functions that existed prior to Electron 5.0.0, for example:
```
